cnd/src/swap_protocols/rfc003/actions/bitcoin.rs

```diff
@@ -13,14 +13,14 @@ use crate::{
 use ::bitcoin::{Amount, OutPoint, Transaction};
 use blockchain_contracts::bitcoin::{rfc003::bitcoin_htlc::BitcoinHtlc, witness::PrimedInput};
 
-impl<B> FundAction<B, asset::Bitcoin> for (B, asset::Bitcoin)
+impl<B> FundAction<B, asset::Bitcoin, identity::Bitcoin> for (B, asset::Bitcoin, identity::Bitcoin)
```

Comment:
Why do we need to expand the tuple here?

```suggestion
impl<B> FundAction<B, asset::Bitcoin, identity::Bitcoin> for (B, asset::Bitcoin)
```

The reason this is implemented for a tuple is so we can use `where` clauses with this tuple in other places (like here: https://github.com/comit-network/comit-rs/blob/ede13873527e377e2c9707e6a4a472f968624963/cnd/src/swap_protocols/rfc003/bob/actions/erc20.rs#L19). That is only useful for completely orthogonal parameters (ledgers and assets). The identity is always bound to the ledger and hence shouldn't need to be included in this tuple.